BDArmory/Guidances/MissileGuidance.cs

```diff
@@ -358,6 +359,40 @@ public static Vector3 GetAirToGroundTarget(Vector3 targetPosition, Vector3 targe
             }
         }
 
+        public static Vector3 GetAirToAirHybridTarget(Vector3 targetPosition, Vector3 targetVelocity,
+            Vector3 targetAcceleration, Vessel missileVessel, float termDist, out float timeToImpact,
+            MissileBase.GuidanceModes homingModeTerminal, float N, float minSpeed = 200)
+        {
+            Vector3 velDirection = missileVessel.srf_vel_direction; //missileVessel.Velocity().normalized;
+
+            float targetDistance = Vector3.Distance(targetPosition, missileVessel.transform.position);
+
+            float currSpeed = Mathf.Max((float)missileVessel.srfSpeed, minSpeed);
+            Vector3 currVel = currSpeed * velDirection;
+
+            float leadTime = (float)(1 / ((targetVelocity - currVel).magnitude / targetDistance));
```

Comment:
This simplifies to
```csharp
float leadTime = targetDistance / (targetVelocity - currVel).magnitude;
```
Are there any potential issues if this returns `float.PositiveInfinity` for `timeToImpact`?

Reply:
Not exactly sure why it's written like this, but that's how it's set up in multiple cases throughout the codebase. Should not cause major issues except a re-evaluation of whether or not it should loft. That won't cause major issues. The only time I would expect (targetVelocity - currVel).magnitude = 0 would be if the missile is just coming off the rails or if the missile has already spent all of its energy. In both cases a re-evaluation wouldn't affect anything.